osu.Game/Overlays/Direct/DownloadTrackingComposite.cs

```diff
@@ -120,12 +121,16 @@ private void onRequestFailure(Exception e)
             Schedule(() => attachDownload(null));
         }
 
-        private void setAdded(BeatmapSetInfo s, bool existing, bool silent)
+        private void setAdded(BeatmapSetInfo s, bool existing, bool silent) => setDownloadState(s, DownloadState.Downloaded);
```

Comment:
Shouldn't this be `LocallyAvailable`?

Reply:
yes, it should. will fix this ~10 hours later.